cape/cape_result.py

```diff
@@ -1375,17 +1375,18 @@ def _get_dns_map(
 
         # A DNS pointer record (PTR for short) provides the domain name associated with an IP address.
         if dns_type == "PTR" and "in-addr.arpa" in request:
```

Comment:
```suggestion
        if dns_type == "PTR":
            continue
```

Since we are ignoring all PTR requests, this is all that is needed.

Also we don't need old code commented out in the code base, so you can delete the code that is not used.